.github/workflows/python.yml

```diff
@@ -96,33 +96,61 @@ jobs:
     needs:
       - linting
       - test-app
-    if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop' || startsWith(github.ref, 'refs/tags/')
+    if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop' || (github.event_name == 'release' && github.event.action == 'released')
     steps:
     - uses: actions/checkout@v3
+    - uses: docker/setup-qemu-action@v2
```

Comment:
Should we specify Qemu VMs?
platforms -> Platforms to install (e.g. arm64,riscv64,arm ; default all)

https://github.com/docker/setup-qemu-action#customizing

Reply:
You are right, maybe it's faster. Done